Source/Scene/WebMapServiceImageryProvider.js

```diff
@@ -174,6 +206,33 @@ define([
         });
     }
 
+    function requestImage(imageryProvider, col, row, level, request, interval) {
+        var dynamicIntervalData = defined(interval) ? interval.data : undefined;
+        var resource = imageryProvider._tileProvider._resource; // We actually want to set the time parameter within the tile provider.
+        var parameters = {};
+        var keys = [];
+        if (defined(dynamicIntervalData)) {
+            if (dynamicIntervalData instanceof Object){
+                try {
+                    Object.keys(dynamicIntervalData).forEach(function (key, index) {
+                        parameters[key] = dynamicIntervalData[key];
+                        keys.push(key.toLowerCase());
+                    });
+                } catch (err){
+                    // Warn the user, this may not be a problem
+                    console.warn('Data from interval has problems.', err);
```

Comment:
In Cesium we don't output errors to the console if its a error by the developer. We occasionally use it if there is an error from a remote server. You can search for DeveloperError to see how we use it and also remove it from the minified build. Although I anticipate that this try/catch will go away.